Source/DafnyCore/AST/Expressions/Expressions.cs

```diff
@@ -1711,6 +1711,8 @@ public TypeTestExpr(IToken tok, Expression expr, Type toType)
 }
 
 public class BinaryExpr : Expression, ICloneable<BinaryExpr> {
+  public bool SecondArgumentIsDomainOfQuantifiedVariable { get; set; }
+  public bool SecondArgumentIsQuantifiedVariableRange { get; set; }
```

Comment:
I'm all in to reduce the number of tokens, but now this information is not cloned if the BinaryExpr is clone. How do you ensure that the behavior is the same?

Reply:
Fixed